script/config.py

```diff
@@ -17,3 +17,5 @@
 GRID5K_MAX_NODE = 50
 
 TOOL_TIMEOUT = "120"
+#Format HH:MM
+GRID5K_TIME_OUT = "2:15"
```

Comment:
This property should be automatically computed from `TOOL_TIMEOUT`

Reply:
Yes, good point. It should take TOOL_TIMEOUT and add it an X%